.gitignore

```diff
@@ -2,3 +2,4 @@ build/
 vendor/
 composer.lock
 .php_cs.cache
+psalm.xml
```

Comment:
any reason to ignore this one?
We must keep this one and rename `psalm.xml.dist` to `psalm.xml`. Dist are for skeletons :)

Reply:
That's exactly the purpose, dist is used as skeleton And when you use the `./vendor/bin/psalm --init` command it generates a `psalm.xml` file which contains absolute paths to this folder
So it must not be shared (you can already see absolute path in the dist, but when the init command is used it is updated for your env)

Reply:
So, I think it must be added in .travis.yml and executed on each pull request. wdyt?

Reply:
@PierreRambaud Ok done! And I included the init in the psalm command ;)